jhotdraw-samples/jhotdraw-samples-misc/src/main/java/org/jhotdraw/samples/svg/SVGView.java

```diff
@@ -50,6 +50,9 @@
      */
     private UndoRedoManager undo;
     private PropertyChangeListener propertyHandler;
+    // Variables declaration - do not modify//GEN-BEGIN:variables
```

Comment:
Why all those `// Variables declaration - do not modify//GEN-BEGIN:variables`? Where does this come from?